services/api/src/resources/project/harborSetup.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+
+/* ::
+import type MariaSQL from 'mariasql';
+*/
+
+const harborClient = require('../../clients/harborClient');
+const logger = require('../../logger');
+const R = require('ramda');
+const Sql = require('../env-variables/sql');
+const { isPatchEmpty, prepare, query, whereAnd } = require('../../util/db');
+
+const lagoonHarborRoute = R.compose(
+  R.defaultTo('http://172.17.0.1:8084'),
+  R.find(R.test(/harbor-nginx/)),
+  R.split(','),
+  R.propOr('', 'LAGOON_ROUTES'),
+)(process.env);
+
+const createHarborOperations = (sqlClient /* : MariaSQL */) => ({
+  addProject: async (lagoonProjectName, projectID) => {
+    // Create harbor project
+    try {
+      var res = await harborClient.post(`projects`, {
+        body: {
+          count_limit: -1,
+          project_name: lagoonProjectName,
+          storage_limit: -1,
+          metadata: {
+            auto_scan: "true",
+            reuse_sys_cve_whitelist: "true",
+            public: "false"
+          }
+        }
+      });
+      logger.debug(`Harbor project ${lagoonProjectName} created!`)
+    } catch (err) {
+      // 409 means project already exists
+      // 201 means project created successfully
+      if (err.statusCode == 409) {
+        logger.info(`Unable to create the harbor project "${lagoonProjectName}", as it already exists in harbor!`)
+      } else {
+        console.log(res)
+        logger.error(`Unable to create the harbor project "${lagoonProjectName}", error: ${err}`)
+      }
+    }
+
+    // Get new harbor project's id
+    try {
+      const res = await harborClient.get(`projects?name=${lagoonProjectName}`)
+      var harborProjectID = res.body[0].project_id
+      logger.debug(`Got the harbor project id for project ${lagoonProjectName} successfully!`)
+    } catch (err) {
+      if (err.statusCode == 404) {
+        logger.error(`Unable to get the harbor project id of "${lagoonProjectName}", as it does not exist in harbor!`)
+      } else {
+        logger.error(`Unable to get the harbor project id of "${lagoonProjectName}" !!`)
+      }
+    }
+
+    console.log(harborProjectID)
```

Comment:
if it's useful to have this in the logs, then can we remove this which would just print a number (?) with no context and add it to the debug() on line 52?

e.g., `logger.debug(`Harbor project id for ${lagoonProjectName}: ${harborProjectId}`)`